.github/workflows/tests.yaml

```diff
@@ -0,0 +1,29 @@
+name: tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  translations-check:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [18, 17, 16, 15, 14, 12, 10, 10.13]
```

Comment:
To be nonbreaking, this needs to test on all currently supported nodes, which includes 11 and 13.

Can i suggest using the same setup I've got here: https://github.com/airbnb/jest-wrap/blob/master/.github/workflows/node-4%2B.yml ?

Reply:
Hopefully I understood everything correctly and moved setup as it should be moved 😄